fe/fe-core/src/test/java/com/starrocks/analysis/CreateMaterializedViewTest.java

```diff
@@ -3683,6 +3683,45 @@ public void testCreateMaterializedViewOnListPartitionTables1() throws Exception
         starRocksAssert.dropTable("list_partition_tbl1");
     }
 
+    @Test
+    public void testAsyncCreateMaterializedViewOnListPartitionTables1WithoutMvPartionExpr() throws Exception {
+        String createSQL = "CREATE TABLE test.list_partition_tbl1 (\n" +
+                "      id BIGINT,\n" +
+                "      age SMALLINT,\n" +
+                "      dt VARCHAR(10),\n" +
+                "      province VARCHAR(64) not null\n" +
+                ")\n" +
+                "ENGINE=olap\n" +
+                "DUPLICATE KEY(id)\n" +
+                "PARTITION BY LIST (province) (\n" +
+                "     PARTITION p1 VALUES IN (\"beijing\",\"chongqing\") ,\n" +
+                "     PARTITION p2 VALUES IN (\"guangdong\") \n" +
+                ")\n" +
+                "DISTRIBUTED BY HASH(id) BUCKETS 10\n" +
+                "PROPERTIES (\n" +
+                "    \"replication_num\" = \"1\"\n" +
+                ")";
+        starRocksAssert.withTable(createSQL);
+
+        LocalDateTime startTime = LocalDateTime.now().plusSeconds(3);
+        String sql = "create materialized view list_partition_mv1 " +
+                "distributed by hash(dt, province) buckets 10 " +
```

Comment:
This should be supported since mv1 is not partitioned.

Also list partitioned mv will be supported later in https://github.com/StarRocks/starrocks/issues/46087

Reply:
thanks 
'This should be supported since mv1 is not partitioned' this scene will result error in getRangePartitionMap